lib/Dialect/Lattigo/IR/LattigoBGVAttributes.td

```diff
@@ -0,0 +1,26 @@
+#ifndef LIB_DIALECT_LATTIGO_IR_LATTIGOBGVATTRIBUTES_TD_
+#define LIB_DIALECT_LATTIGO_IR_LATTIGOBGVATTRIBUTES_TD_
+
+class Lattigo_BGVAttribute<string attrName, string attrMnemonic>
+      : Lattigo_Attribute<"BGV" # attrName, "bgv." # attrMnemonic> {
+    let summary = "Attribute for Lattigo BGV";
+    let description = [{
+        This attribute represents the values for Lattigo BGV.
+    }];
+}
+
+def Lattigo_BGVParametersLiteral
+    : Lattigo_BGVAttribute<"ParametersLiteral", "parameters_literal"> {
+    let summary = "Literal parameters for Lattigo BGV";
+    let description = [{
+        This attribute represents the literal parameters for Lattigo BGV.
+    }];
+    let parameters = (ins
+      "int":$logN,
+      "DenseI32ArrayAttr":$logQ,
```

Comment:
The docs should specify what these parameters mean, or link to Lattigo docs explaining it. For instance, someone might be confused by the discrepancy between the name (suggesting a scalar) and the type (an array)